app/components/widgets/AlertBox.vue.ts

```diff
@@ -203,4 +208,18 @@ export default class AlertBox extends WidgetSettings<IAlertBoxData, AlertBoxServ
     this.save();
     this.editingName = null;
   }
+
+  handleFacemaskInput() {
+    if (this.selectedAlert === 'facemasks') {
+      const { duration } = this.selectedVariation.settings;
+      this.facemasksService
+        .updateFacemaskSettings({
+          duration,
+          enabled: this.facemaskEnabled,
+          device: this.facemasksService.getEnabledDevice(),
+        })
+        .catch(() => this.onFailHandler($t('Something went wrong updating Facemask settings')));
```

Comment:
Do we still see value in reporting to Sentry? 
`Sentry.captureException` prior to invoking `onFailHandler`? Please disregard if the handler already does this.

Reply:
i dont think we report to sentry on this yet, but would be a good thing to add in, i'll make a future PR with that